### PR TITLE
gh-119689: generate stack effect metadata for pseudo instructions

### LINKED ISSUE #119689: Generate stack effect for pseudo instructions
The opcode metadata has stack effect for real instructions, and then in code we add the pseudo instructions. Would be tidier to generate it for the pseudo instructions as well.

### Linked PRs
* gh-119691

### PR DESCRIPTION
**Pull request by iritkatriel**
Fixes #119689.

* Issue: gh-119689

### CODE REVIEW

File: Python/compile.c [hunk 1]
```diff
@@ -703,51 +703,22 @@ compiler_set_qualname(struct compiler *c)
 static int
 stack_effect(int opcode, int oparg, int jump)
```
Comment (gvanrossum) [inline Python/compile.c:704]: Is `jump` still used? The deleted `SETUP` cases use it, but I don't think the generated code uses it. Is it no longer needed? Could you remove the `jump` argument then? (All the callers are inside this file.)

**Review by gvanrossum — APPROVED** [review]
LGTM, though I'm not sure what should happen to the `jump` argument.

File: Python/compile.c (hunk 1 above)
Comment (iritkatriel) [reply Python/compile.c:704]: jump is exposed in the c api and used from flowgraph.c.

Comment (gvanrossum) [reply Python/compile.c:704]: I see, the key is `IS_BLOCK_PUSH_OPCODE(opcode)`. Sorry I didn't get that at first.

**Pull request merged**